lessons/lesson-template.md

```diff
@@ -0,0 +1,74 @@
+- title: 
+- tags: 
+
+###Standards:
```

Comment:
Can we use h1 / `#` for the top-level sections, and h2 / `##` for subsections for instance in the lecture section?